auacm/app/static/html/scoreboard.html

```diff
@@ -11,7 +11,7 @@ <h2 class="pull-right" ng-show="active">{{ timeLeft | secondsToDateTime | date:"
           <th>Team</th>
           <th>Solved</th>
           <th>Time</th>
-          <th ng-repeat="name in problemNames">{{ name }}</th>
+          <th ng-repeat="name in problemNames"><a href="/#/problems/{{ problemData[name].shortname }}">{{ name }}</a></th>
```

Comment:
This link isn't working for me; it just goes back to the problem page. I'm looking into it now